package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "hw",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "lint": "biome check .",
+    "format": "biome format . --write",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "19.0.0",
+    "react-native": "0.79.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.25.2",
+    "@babel/preset-env": "^7.25.3",
+    "@babel/runtime": "^7.25.0",
+    "@eslint/js": "^9.26.0",
+    "@react-native-community/cli": "18.0.0",
+    "@react-native-community/cli-platform-android": "18.0.0",
+    "@react-native-community/cli-platform-ios": "18.0.0",
+    "@react-native/babel-preset": "0.79.2",
+    "@react-native/eslint-config": "0.79.2",
+    "@react-native/metro-config": "0.79.2",
+    "@react-native/typescript-config": "0.79.2",
+    "@types/jest": "^29.5.13",
+    "@types/react": "^19.0.0",
+    "@types/react-test-renderer": "^19.0.0",
+    "biome": "^0.3.3",
+    "eslint": "^8.57.1",
```

Comment:
Ter tanto o biome quanto o eslint fica redundante e pode até ir contra o propósito de usar um linter no projeto, já que eles fazem a mesma coisa e o código pode ficar com um padrão diferente dependendo de qual for utilizado.

Poderia manter só o eslint + prettier nesse caso, tirar o biome das dependências e apagar o biome.json